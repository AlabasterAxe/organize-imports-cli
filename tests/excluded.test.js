const runCli = require("./run-cli");

describe("exclude in tsconfig", () => {
  runCli("cli/exclude", ["included.ts", "excluded.ts"]).test();
});
