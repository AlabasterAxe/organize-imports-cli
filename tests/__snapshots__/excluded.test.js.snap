// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exclude in tsconfig (status) 1`] = `0`;

exports[`exclude in tsconfig (stderr) 1`] = `""`;

exports[`exclude in tsconfig (stdout) 1`] = `
"Organizing imports...
<dir>/included.ts/*CR*/
<dir>/included.ts (modified)
<dir>/excluded.ts/*CR*/
<dir>/excluded.ts (modified)
Done!
"
`;

exports[`exclude in tsconfig (write) 1`] = `
Array [
  Object {
    "content": "import { log } from \\"./util\\";

log(\\"bar\\");
",
    "filename": "included.ts",
  },
  Object {
    "content": "import { log } from \\"./util\\";

log(\\"foo\\");
",
    "filename": "excluded.ts",
  },
]
`;
